core/services/chainlink/config_pytoscope.go

```diff
@@ -0,0 +1,23 @@
+package chainlink
+
+import v2 "github.com/smartcontractkit/chainlink/v2/core/config/v2"
+
+type PyroscopeConfig struct {
```

Comment:
@george-dorin Do we need to export this? So far I've been keeping these sub-configs private as everything refers to the interface rather than the concrete type.

Reply:
You are right, there is no reason to export, updated in [`7ad3319` (#9476)](https://github.com/smartcontractkit/chainlink/pull/9476/commits/7ad3319233f1e716ff16263e541c46a89e5fe0ce)